test/kreflection/test-kreflection.sh

```diff
@@ -0,0 +1,9 @@
+TOP=$(git rev-parse --show-toplevel)
+KRUN=$TOP/.build/k/bin/krun
+KORE_EXEC=$TOP/.build/kore/bin/kore-exec
+KRUN_OPTS="--haskell-backend-command \"$KORE_EXEC --disable-log-timestamps\""
+COMMAND="$KRUN $KRUN_OPTS -d . 1.kreflection 2>test-kreflection.sh.out || true"
+KOMPILE="$TOP/.build/k/bin/kompile kreflection.k --backend haskell"
+eval $KOMPILE
```

Comment:
You should not run `kompile` here, but let `make` handle the compilation by including a dependency on `KOMPILED`.